pkgs/applications/networking/irc/convos/default.nix

```diff
@@ -67,16 +69,24 @@ perlPackages.buildPerlPackage rec {
       > t/00_nixpkgs_module_install.t
   '';
 
+  # (temp) bump the version - see https://github.com/convos-chat/convos/issues/858
+  preInstall = ''
+    substituteInPlace lib/Convos.pm \
+      --replace '7.15' '7.16'
+    substituteInPlace script/convos \
+      --replace '7.15' '7.16'
+  '';
```

Comment:
Is this still needed for v8.x?

Reply:
Yeah this can be dropped, thanks for reminding me! 👍